dbaas-cli/cmd/mongo/create.go

```diff
@@ -102,6 +102,10 @@ var createCmd = &cobra.Command{
 					log.WithField("database", cluster).Info("information")
 					return
 				}
+			case "error":
```

Comment:
Consider making all these states as typed constants.

Reply:
https://github.com/Percona-Lab/percona-dbaas-cli/pull/101/commits/600ae3a23e647cb3f5ddb2e64960c6ca1f41050f

Reply:
I meant **_typed_** constants.
https://github.com/Percona-Lab/percona-dbaas-cli/blob/600ae3a23e647cb3f5ddb2e64960c6ca1f41050f/dbaas-lib/structs/structs.go#L12
Status shouldn't be a string but a user type like in operators.
The same I believe should be done with Engine and Provider.

And, btw, maybe all this "structs" package should be part of `github.com/Percona-Lab/percona-dbaas-cli/dbaas-lib/dbaas`?

Reply:
https://github.com/Percona-Lab/percona-dbaas-cli/pull/101/commits/7ce73ab37cf608c44c28315bf025ea3a3746a2e1